src/averageAges.js

```diff
@@ -55,7 +76,22 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const child = !onlyWithSon
+    ? people.filter(item => {
+      return people.find(elem => elem.name === item.mother);
+    })
+
+    : people.filter(item => {
+      return people.find(elem => elem.name === item.mother && item.sex === 'm');
+    });
+
+  // eslint-disable-next-line max-len
+  const ageDif = child.map(elem => elem.born - people.find(item =>
+    item.name === elem.mother).born);
```

Comment:
```suggestion
  const ageDif = child.map(elem => elem.born - people.find(item =>
    item.name === elem.mother).born);
```
do not disable the linter, just format your code correctly